lib/basedriver/driver.js

```diff
@@ -30,6 +32,12 @@ const ON_UNEXPECTED_SHUTDOWN_EVENT = 'onUnexpectedShutdown';
 
 class BaseDriver extends Protocol {
 
+  /**
+   * Make the basedriver version available so for any driver which inherits from this package, we
+   * know which version of basedriver it inherited from
+   */
+  static baseVersion = BASEDRIVER_VER;
```

Comment:
is `static` supported in ECMA standard that Appium uses?

Reply:
it was already in the list of our babel transpilation plugins. i merely had to add it to our linter as well.